src/stories/containers/cu-table/cu-table-filters.tsx

```diff
@@ -42,6 +43,19 @@ export const Filters = (props: FilterProps) => {
     });
   }, [router]);
 
+  const handleCloseSearch = () => {
+    const search = router.query;
+    search.searchText = '';
+    router.push({
+      pathname: '/',
+      search: stringify(search),
+    });
+    const input = document.querySelector('#search-input');
```

Comment:
Please, avoid using direct DOM manipulation every time it is possible. In this case, you can use `useRef` to refer to the search input and just handle the input value in the React way instead. Also, be sure you only add the `searchText` query param when it has value to keep a clean URL

Reply:
you are right , i will use the correct way